sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -1,13 +1,227 @@
-# Azure Opentelemetry Exporter for Monitor
+# Microsoft Opentelemetry exporter for Azure Monitor
+
+[![Gitter chat](https://img.shields.io/gitter/room/Microsoft/azure-monitor-python)](https://gitter.im/Azure/azure-sdk-for-python)
+
+The exporter for Azure Monitor allows you to export tracing data utilizing the OpenTelemetry SDK and send telemetry data to Azure Monitor for applications written in Python.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor) | [Package (PyPi)][pypi] | [API reference documentation][api_docs] | [Product documentation][product_docs] | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/samples) | [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/CHANGELOG.md)
+
+> **NOTE**: This is the preview for the next major version of the [`opentelemetry-azure-monitor-python`](https://github.com/microsoft/opentelemetry-azure-monitor-python).
 
 ## Getting started
 
+### Install the package
+
+Install the Microsoft Opentelemetry Exporter Azure Monitor client library for Python with [pip][pip]:
+
+```Bash
+pip install microsoft-opentelemetry-exporter-azuremonitor --pre
+```
+
+### Prerequisites: 
+To use this package, you must have:
+* Azure subscription - [Create a free account][azure_sub]
+* Azure Monitor - [How to use application insights][application_insights_namespace]
+* Opentelemetry SDK - [Opentelemtry SDK for Python][ot_sdk_python]
+* Python 3.5 or later - [Install Python][python]
+
+### Authenticate the client
+
+Interaction with Azure monitor exporter starts with an instance of the `AzureMonitorSpanExporter` class. You will need an **instrumentation_key** or a **connection_string** to instantiate the object.
```

Comment:
If we are moving towards the use of connection string, would it be better to omit anything related to instrumentation key and only recommend connection string?

Reply:
I agree with that tbh! There should be one - and preferably only one obvious way to do it :) 
Let's stop documenting it in the readme for this beta but still support it, that way we dont break much stuff on the day of the release but will not encourage users to use it.